packages/framework/esm-styleguide/src/diagnoses-tags/diagnosis-tags.component.tsx

```diff
@@ -0,0 +1,30 @@
+import { Tag } from '@carbon/react';
+import { type Diagnosis } from '@openmrs/esm-api';
+import React from 'react';
+import { useDiagnosisRankToColorMap } from './diagnosis-tags.resource';
+
+interface DiagnosisTagsProps {
+  diagnoses: Diagnosis[];
```

Comment:
Style-wise we've preferred this syntax. They're equivalent, but consistency is important.

```suggestion
  diagnoses: Array<Diagnosis>;
```